persistence/test/gov_test.go

```diff
@@ -31,8 +32,175 @@ func TestGetSetParam(t *testing.T) {
 	err = db.SetParam(types.AppMaxChainsParamName, newMaxChains)
 	require.NoError(t, err)
 
-	maxChains, err := db.GetMaxAppChains()
+	height, err := db.GetHeight()
+	require.NoError(t, err)
+
+	maxChains, err := db.GetIntParam(types.AppMaxChainsParamName, height)
+	require.NoError(t, err)
+
+	require.Equal(t, newMaxChains, maxChains)
+}
+
+func TestGetSetStringParam(t *testing.T) {
+	db := persistence.PostgresContext{
+		Height:     0,
+		PostgresDB: testPostgresDB,
+	}
+
+	err := db.InitParams()
+	require.NoError(t, err)
+
+	newServiceNodeMinimumStake := "99999999"
+
+	err = db.SetParam(types.ServiceNodeMinimumStakeParamName, newServiceNodeMinimumStake)
+	require.NoError(t, err)
+
+	height, err := db.GetHeight()
+	require.NoError(t, err)
+
+	serviceNodeMinimumStake, err := db.GetStringParam(types.ServiceNodeMinimumStakeParamName, height)
+	require.NoError(t, err)
+
+	require.Equal(t, newServiceNodeMinimumStake, serviceNodeMinimumStake)
+}
+
+func TestGetSetByteArrayParam(t *testing.T) {
+	db := persistence.PostgresContext{
+		Height:     0,
+		PostgresDB: testPostgresDB,
+	}
+
+	err := db.InitParams()
+	require.NoError(t, err)
+
+	newOwner, _ := hex.DecodeString("=cXYUx2RvJnWjV1duBHahBlcwNFU4ZWathTU2czc0QnMYxWYwgESSxWdNJkLw4WS5AjealmWIJGSSdUY3hmVZpnWVJmaCRlTYBHbhdFZrdlbOdlYpJESjllRyM2RxITW3VleWFGctZFSwFjW2J1aW1EbUJlVKl2Tp9GSilXTHFGNxc1VzpUMSJlS5VmL5o0QWhFcrlkNJN0Y1IlbJNXSp5UMJpXVJpUaPl2YHJGaKlXZ")
```

Comment:
I tried to decode it but got an error. Not sure if I'm missing something.

Might be worth doing a `require.NoError(t, err)` on the response because otherwise `newOwner` is an empty slice.


<img width="499" alt="Screen Shot 2022-08-08 at 7 12 24 PM" src="https://user-images.githubusercontent.com/1892194/183530057-f0449c07-1639-4ed7-99d8-4b70df5652f9.png">

Reply:
Ops! Genuine mistake 🤦‍♂️ but good that it allowed us to spot the missing `require.NoError(t, err)`